### PR TITLE
Proofreading days 1-4. Suggestions enumerated below:

### PR DESCRIPTION
**Pull request by leifwickland**
- Correct typos and misspellings
- Insert definite and indefinite articles
- Consistently enclose type names in backticks in headings
- Attempt to apply consistent capitalization to "_x_ laws"
- Apply punctuation rules for tiles and words used per se

### CODE REVIEW

File: docs/00.md [hunk 1]
```diff
@@ -18,13 +18,13 @@ This is a log of me going through Cats, a functional programming library for Sca
 that is currently *experimental* and under active development.
 I'll try to follow the structure of [learning Scalaz][lsz] that I wrote in 2012 (!).
 
-Cats [website][cats] says the name is a playful shortening of the word category.
-They say coordinating programmers are like herding cats
-(as in attempting to move dozens of them in a direction like cattles, while cats have their own ideas),
-and it seems to be true at least for those of us using Scala. Keenly aware of this situation, cats list
-approachability as their first motivation.
+Cats' [website][cats] says the name is a playful shortening of the word category.
+It says coordinating programmers is like herding cats
```
Comment (eed3si9n) [inline docs/00.md:22]: "They" here refers to unnamed people, not the website.

File: docs/00.md [hunk 2]
```diff
@@ -18,13 +18,13 @@ This is a log of me going through Cats, a functional programming library for Sca
...
+(as in attempting to move dozens of them in a direction like cattle, while cats have their own ideas),
+and it seems to be true at least for those of us using Scala. Keenly aware of this situation, Cats' site lists
```
Comment (eed3si9n) [inline docs/00.md:24]: This was a play on word, by me calling the library authors as cats.

File: docs/00.md (hunk 1 above)
Comment (eed3si9n) [inline docs/00.md:22]: Would "Some people say" make it clearler?

File: docs/02/a.md [hunk 3]
```diff
@@ -24,7 +24,7 @@ The conventional steps of defining a modular typeclass in Scala used to look lik
 
 Frankly, these steps are mostly copy-paste boilerplate except for the first one.
 Enter Michael Pilquist ([@mpilquist][@mpilquist])'s [simulacrum][simulacrum].
-simulacrum magically generates most of steps 2-4 just by putting `@typeclass` annotation.
+Simulacrum magically generates most of steps 2-4 just by putting `@typeclass` annotation.
```
Comment (eed3si9n) [inline docs/02/a.md:27]: For proper nouns like project names shouldn't we stick to one casing? It should be simulacrum (all lowercase) everywhere or Simulacrum (capitalized) everywhere.

File: docs/01/b.md [hunk 4]
```diff
@@ -2,13 +2,13 @@
   [algebra]: https://github.com/non/algebra
   [Equivalence]: http://en.wikipedia.org/wiki/Equivalence_relation
 
-## Eq
+## `Eq`
```
Comment (eed3si9n) [inline docs/01/b.md:5]: I think one of the reasons I didn't do this was how it looks as a header or on toc. Let me try out this PR locally.

Comment (eed3si9n) [inline docs/01/b.md:5]: Yea the toc gets messed up because of a Pamflet bug 

screnshot

File: docs/00.md [hunk 5]
```diff
@@ -18,13 +18,13 @@ This is a log of me going through Cats, a functional programming library for Sca
 that is currently *experimental* and under active development.
 I'll try to follow the structure of [learning Scalaz][lsz] that I wrote in 2012 (!).
 
-Cats [website][cats] says the name is a playful shortening of the word category.
-They say coordinating programmers are like herding cats
-(as in attempting to move dozens of them in a direction like cattles, while cats have their own ideas),
-and it seems to be true at least for those of us using Scala. Keenly aware of this situation, cats list
-approachability as their first motivation.
+Cats' [website][cats] says the name is a playful shortening of the word category.
```
Comment (eed3si9n) [inline docs/00.md:21]: The Chicago style would put _'s_ after the possessive even when the proper noun ends with _s_, so you're already in on treating Cats as plural ;)

File: docs/01/b.md [hunk 6]
```diff
@@ -2,13 +2,13 @@
...
 
 LYAHFGG:
 
 > `Eq` is used for types that support equality testing. The functions its members implement are `==` and `/=`.
 
-Cats equivalent for the `Eq` typeclass is also called `Eq`.
+Cats' equivalent for the `Eq` typeclass is also called `Eq`.
```
Comment (eed3si9n) [inline docs/01/b.md:11]: I'd adopt Chicago style here and make it Cats's.

File: docs/01/c.md [hunk 7]
```diff
@@ -1,11 +1,11 @@
 
-### Order
+### `Order`
 
 LYAHFGG:
 
 > `Ord` is for types that have an ordering. `Ord` covers all the standard comparing functions such as `>`, `<`, `>=` and `<=`.
 
-Cats equivalent for the `Ord` typeclass is `Order`.
+Cats' equivalent for the `Ord` typeclass is `Order`.
```
Comment (eed3si9n) [inline docs/01/c.md]: Chicago.

File: docs/01/e.md [hunk 8]
```diff
@@ -1,11 +1,11 @@
 
-### Show
+### `Show`
 
 LYAHFGG:
 
 > Members of `Show` can be presented as strings.
 
-Cats equivalent for the `Show` typeclass is `Show`:
+Cats' equivalent for the `Show` typeclass is `Show`:
```
Comment (eed3si9n) [inline docs/01/e.md]: Windy city.

File: docs/01/f.md [hunk 9]
```diff
@@ -1,11 +1,11 @@
 
-### Read
+### `Read`
 
 LYAHFGG:
 
 > `Read` is sort of the opposite typeclass of `Show`. The `read` function takes a string and returns a type which is a member of `Read`.
 
-I could not find Cats equivalent for this typeclass.
+I could not find Cats' equivalent for this typeclass.
```
Comment (eed3si9n) [inline docs/01/f.md:8]: Home of Cubs.

File: docs/01/g.md [hunk 10]
```diff
@@ -2,13 +2,13 @@
...
 
-### Enum
+### `Enum`
 
 LYAHFGG:
 
 > `Enum` members are sequentially ordered types — they can be enumerated. The main advantage of the `Enum` typeclass is that we can use its types in list ranges. They also have defined successors and predecessors, which you can get with the `succ` and `pred` functions.
 
-I could not find Cats equivalent for this typeclass.
+I could not find Cats' equivalent for this typeclass.
```
Comment (eed3si9n) [inline docs/01/g.md]: Chi-town.

File: docs/01/h.md [hunk 11]
```diff
@@ -1,13 +1,13 @@
...
 
-### Numeric
+### `Numeric`
 
 LYAHFGG:
 
 > `Num` is a numeric typeclass. Its members have the property of being able to act like numbers.
 
-I could not find Cats equivalent for this typeclass,
+I could not find Cats' equivalent for this typeclass,
```
Comment (eed3si9n) [inline docs/01/h.md]: Chicago.

File: docs/00.md (hunk 2 above)
Comment (leifwickland) [inline docs/00.md:24]: How about "...this situation, the Cats' cats list..."?

File: docs/00.md (hunk 1 above)
Comment (leifwickland) [inline docs/00.md:22]: Yeah, I think that does clear it up.

File: docs/02/a.md (hunk 3 above)
Comment (leifwickland) [inline docs/02/a.md:27]: You're absolutely right. Sorry about that.

File: docs/00.md (hunk 2 above)
Comment (eed3si9n) [inline docs/00.md:24]: Putting `the` does make it grammatically correct. `Cats's cats` kills the fun. `the cats` maybe.

Comment (leifwickland) [inline docs/00.md:24]: Done.

File: docs/00.md (hunk 5 above)
Comment (eed3si9n) [inline docs/00.md:21]: Here's a link on this topic http://www.chicagomanualofstyle.org/qanda/data/faq/topics/PossessivesandAttributives.html:

> Q. When indicating possession of a word that ends in s, is it correct to repeat the s after using an apostrophe? For example, which is correct: “Dickens’ novel” or “Dickens’s novel”?
> 
> A. Either is correct, though we prefer the latter. Please consult 7.15–18 for a full discussion of the rules for forming the possessive of proper nouns. For a discussion of the alternative practice of simply adding an apostrophe to form the possessive of proper nouns ending in s, see paragraph 7.21.

Comment (eed3si9n) [inline docs/00.md:21]: Actually I'm wrong on this for Chicago Style due to exception clause 7.19 _Nouns plural in form, singular in meaning_.

File: docs/01/b.md (hunk 4 above)
Comment (leifwickland) [inline docs/01/b.md:5]: You previously had backticks on fourth level headers in some places. Do you want me to make that consistent or leave it alone? Here're the real questions: do you want backticks on any headers? If so, from what level on?

File: docs/00.md (hunk 5 above)
Comment (leifwickland) [inline docs/00.md:21]: Is the summary of the above that you're fine with `Cats'`?

File: docs/01/b.md (hunk 4 above)
Comment (eed3si9n) [inline docs/01/b.md:5]: Probably no backticks throughout would be most consistent.

File: docs/00.md (hunk 5 above)
Comment (eed3si9n) [inline docs/00.md:21]: Yes, you are right and I was wrong.

Comment (leifwickland) [inline docs/00.md:21]: Let's go with "I was lucky to ignorantly strike upon one style style guide's preference." I'll make it consistent to the best of my ability.

File: docs/01/b.md (hunk 6 above)
Comment (eed3si9n) [inline docs/01/b.md:11]: Chicago fail. Must find some other manual.

File: docs/01/b.md (hunk 4 above)
Comment (leifwickland) [inline docs/01/b.md:5]: No backticks in headings! Aye aye, cap'n!

File: docs/01/b.md (hunk 6 above)
Comment (leifwickland) [inline docs/01/b.md:11]: https://owl.english.purdue.edu/owl/resource/621/01/

`add 's to the singular form of the word (even if it ends in -s):
the owner's car 
James's hat (James' hat is also acceptable. For plural, proper nouns that are possessive, use an apostrophe after the 's': "The Eggleses' presentation was good." The Eggleses are a husband and wife consultant team.)`

If we assume that "Cats" is singular, we could just go with `Cats's`

Comment (eed3si9n) [inline docs/01/b.md:11]: Nah. Chicago's exception for plural form nouns makes sense.

File: docs/00.md (hunk 5 above)
Comment (leifwickland) [inline docs/00.md:21]: http://en.wikipedia.org/wiki/Links_%28web_browser%29
I endorse how the Wikipedia article on the Links browser treats the plural word as consistently singular proper name.

**Comment by eed3si9n** [comment]
closing this in favor of #2

**Pull request closed without merging**